README.md

```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
 
 ## Labeling a Video
 
-Coming soon!
+Labeling a video is much like labeling a series of images. When a video is selected from the left, it will begin automatically playing, and there are several controls on the player, as seen here:
+![alt text](docs/images/video-player.png "Video Player")
+
+In addition to the normal video playback controls, there are two extra pairs of buttons. 
+
+On the left, there are the previous and next frame buttons. Clicking these will pause the video, and move to the next appropriate frame as determined by the project settings. For example, if the project settings have a frame extraction rate of 1, these buttons will cause the video to be moved back or forward 1 second, where if the frame extraction rate was 10, the video will be moved back or forward a tenth of a second.
+
+On the right, there are the previous and next tagged frame buttons. Clicking these will pause the video and move to the next or previous frame that has a previously tagged region on it, if a tagged frame exists.
+
+Colored lines will also be visible along the video's timeline. These indicate the video frames that have been already been visited. A yellow line denotes a frame that has been visited only, while a green line denotes a frame that has been both visited and tagged. The bars can be clicked for quick navigation to the indicated frame.
```

Comment:
2nd sentence, typo: "...frames that have already been visited"
4th sentence: "The colored lines can be clicked..."
It sounds repetitive to keep saying colored lines, but I think it's clearer.